### PR TITLE
streamlined text descriptions

### PR DESCRIPTION
**Pull request by reyg3**

### CODE REVIEW

**Review by lboeman — CHANGES_REQUESTED** [review]
Needs work

**Comment by reyg3** [comment]
u do

File: README.md [hunk 1]
```diff
@@ -17,28 +17,32 @@
...
 
-The research applications group is led by [Rey Granillo](https://github.com/reyg3) Director of Technology and Research Computing, [Leland Boeman](https://github.com/lboeman) Research & Development Software Engineer, and [Thomas Weiss](https://github.com/tweissaz) Research & Development Systems Engineer.
+TRAD is led by [Rey Granillo](https://github.com/reyg3) Director of Technology and R&D Engineering, [Leland Boeman](https://github.com/lboeman) Research & Development Software Engineer, and [Thomas Weiss](https://github.com/tweissaz) Research & Development Systems Engineer.
 
 ### Our proficiencies
-- Research focused software development
-- Cloud computing
+- Research & Development / Proof of Concept
+  - Research focusd software development
```
Comment (tweissaz) [inline README.md]:
```suggestion
  - Research-focused software development
```

File: README.md [hunk 2]
```diff
@@ -51,7 +55,7 @@ Our current student cohort includes:
   - Majors: Ecology & Evolutionary Biology and Optical Sciences & Engineering
 
 ## Research Collaborators
-In addition to our core research applications development team, we've had a number of collaborators across AIR programs and affiliated groups across the University of Arizona. 
+In addition to our core research team, we've had a number of collaborators across AIR programs and affiliated groups across the University of Arizona.
```
Comment (tweissaz) [inline README.md:58]: Maybe with some upcoming projects we also mention outside collaborators?

**Review by tweissaz — APPROVED** [review]
Think these changes sound better and provide a more streamlined overview of the work we do. We'll want to also make sure this text gets adjusted on the HTML side too.

File: README.md (hunk 2 above)
Comment (reyg3) [reply README.md:58]: I think that's a good idea. We can list ALERTWest and B2. Are there others you can think of that we should highlight @tweissaz ?

Comment (tweissaz) [reply README.md:58]: Nah, all I can think of for now.

File: README.md [hunk 3]
```diff
@@ -17,28 +17,32 @@
...
+The Technical Research Applications Development (TRAD) group is found within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). TRAD is a group of technology research professionals and students who provide a variety of technical research services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within AIR. Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between.
 
-The research applications group is led by [Rey Granillo](https://github.com/reyg3) Director of Technology and Research Computing, [Leland Boeman](https://github.com/lboeman) Research & Development Software Engineer, and [Thomas Weiss](https://github.com/tweissaz) Research & Development Systems Engineer.
+TRAD is led by [Rey Granillo](https://github.com/reyg3) Director of Technology and R&D Engineering, [Leland Boeman](https://github.com/lboeman) Research & Development Software Engineer, and [Thomas Weiss](https://github.com/tweissaz) Research & Development Systems Engineer.
```
Comment (lboeman) [inline README.md]: It feels a little weird to me to abbreviate R&D here  in the first title, then spell it out for Thomas and I.

File: README.md [hunk 4]
```diff
@@ -17,28 +17,32 @@
...
+The Technical Research Applications Development (TRAD) group is found within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). TRAD is a group of technology research professionals and students who provide a variety of technical research services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within AIR. Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between.
```
Comment (lboeman) [inline README.md]:
```suggestion
The Technical Research Applications Development (TRAD) is a group of technology research professionals and students who provide technical research services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between.

```
 Maybe a nit, but I felt like the first sentence didn't say anything previously. If we're trying to answer the question "What is TRAD?" answering with "TRAD is in AIR" first is a little strange. I think this rework gets to the point a little faster.  What do you think?

File: README.md [hunk 5]
```diff
@@ -17,28 +17,32 @@
...
+By developing and combining technologies, we incorporate novel technical solutions into new and existing research. Using a collaborative model, we can better understand project data, the research questions being asked, and provide higher quality research collaborations. Having that greater understanding has led us to find correlations in data that were not previously realized which have led to new research project ideas and proposals. We collaborate with researchers during the proposal process and can provide a proof-of-concept application or tool showing what is technically feasible if project funding is awarded. Upon project completion, we aid with the technical writing of publications which outline our findings and technical solutions that were implemented.
```
Comment (lboeman) [inline README.md]:
```suggestion
Using a collaborative model, TRAD members gain a broader understanding of the research questions being asked, and collaborators gain a better understanding of how cutting edge technology can be applied to their area of expertise. These collaborations have uncovered correlations in data that were not previously realized which have led to new research project ideas and proposals. We collaborate at each step of the research process from proposal development to publication. TRAD can provide a proof-of-concept application or tool showing what is technically feasible to add a competitive edge to proposals. Upon project completion, we aid with the technical writing of publications which outline our findings and technical solutions that were implemented.
```
What do you think about this rework? I felt like some of the points read a little awkwardly and tried to make it more direct.

**Review by lboeman — COMMENTED** [review]
I left some of suggestions for reworks. I don't have any big issues with how things are currently written, so these don't need to be accepted as is or at all if there are objections. Happy to elaborate on changes if needed.

File: README.md (hunk 3 above)
Comment (reyg3) [reply README.md]: Good catch. Removed the abbreviation for consistency.

**Review by lboeman — APPROVED** [review]

**Pull request merged**